src/libstd/sys/common/backtrace.rs

```diff
@@ -36,7 +36,7 @@ pub fn log_enabled() -> bool {
     }
 
     let val = match env::var_os("RUST_BACKTRACE") {
-        Some(..) => 2,
+        Some(x) => if x.eq("disabled") { 1 } else { 2 },
```

Comment:
On another note, if I wanted to use `to_lowercase()` looks like this would work:

```rust
//        Some(x) => if x.eq("disabled") { 1 } else { 2 },
        Some(x) => {
            if match x.to_str() {
                None => "",
                Some(y) => y,
            }.to_lowercase() == "disabled" { 1 } else { 2 }
        },
```

Here's a [test program](https://play.rust-lang.org/?code=use%20std%3A%3Aenv%3B%0A%0Afn%20main()%20%7B%0A%20%20%20%20let%20b%20%3D%20match%20env%3A%3Avar_os(%22HOME%22)%20%7B%0A%20%20%20%20%20%20%20%20None%20%3D%3E%202%2C%0A%20%20%20%20%20%20%20%20Some(x)%20%3D%3E%20%7B%0A%20%20%20%20%20%20%20%20%20%20%20%20if%20match%20x.to_str()%20%7B%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20None%20%3D%3E%20%22%22%2C%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20Some(y)%20%3D%3E%20y%2C%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20.to_uppercase()%20%3D%3D%20%22%2FHOME%2FRUST%22%20%7B%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%201%0A%20%20%20%20%20%20%20%20%20%20%20%20%7D%20else%20%7B%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%202%0A%20%20%20%20%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%3B%0A%20%20%20%20println!(%22%7B%3A%3F%7D%22%2C%20b)%3B%0A%7D%0A&version=stable) that shows the above works.

Should I make a new commit with this? Is this desirable? Maybe there's a better way? Or is not desirable to have this logic(of allowing case insensitive `disabled` to work) ?

Please advise? Thank ye:)